service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java

```diff
@@ -19,4 +21,8 @@
     public TaskType getTaskType() {
         return TaskType.GET_GPC_DOCUMENT;
     }
+
+    public static String extractIdFromUrl(String url) {
+        return StringUtils.substring(url, StringUtils.lastIndexOf(url, "/") + 1);
```

Comment:
Slash could be moved to a constant. Thanks to this instead of recreating it every time the method is invoked, we would use the already established String object.

Reply:
If you create a `PATH_DELIMITER="/"` it becomes more obvious that this method is looking for the last path delimiter, the end of the URL